_contentTemplates/dropdowns/templates.md

```diff
@@ -0,0 +1,15 @@
+#item-template
```

Comment:
We should mention that the template `RenderFragment` exposes a `context` and explain it's the data item object.